utensor_cgen/pbparser.py

```diff
@@ -85,6 +85,14 @@ def parse_pb(file_or_path, output_nodes=None):
   graph_def.ParseFromString(fid.read())
   fid.close()
 
+  if quantize_enabled:
+    from tensorflow.tools.graph_transforms import TransformGraph
+    if output_nodes is None:
+      target_output_nodes = []
+    # Target output nodes expects a list of targets
+    graph_def = TransformGraph(graph_def, [],
+                               target_output_nodes, ["quantize_weights", "quantize_nodes"])
```

Comment:
You need to add `"sort_by_execution_order"` to the `transforms` argument.
Or the output `graph_def` can not be imported properly with `tf.import_graph_def`

Reply:
fixed